### PR TITLE
Fix #1727 -- Retire django-developers and django-users mailing list

### LINKED ISSUE #1727: Retire django-developers and django-users mailing list
Following the forum thread ([Proposal: retiring django-users and django-developers mailing lists](https://forum.djangoproject.com/t/proposal-retiring-django-users-and-django-developers-mailing-lists/35747)), we can now move ahead with updating the website to remove or contextualize references to the django-developers and django-users mailing list. Corresponding Django ticket: [Retire django-developers and django-users mailing list django#35908](https://code.djangoproject.com/ticket/35908)

## Tasks

Here are relevant changes I think we should go ahead with:

- [x] In the footer, remove the `Django Users Mailing List` item
- [x] On the [Community](https://www.djangoproject.com/community/) page, remove the "Mailing Lists" section
- [x] On [Download / How to get Django](https://www.djangoproject.com/download/), replace "be sure to sign up for the django-users mailing list" with "be sure to sign up to the Django Forum"
- [x] On the [homepage](https://www.djangoproject.com/), remove the "Stay in the loop" section
- [ ] CoC-related pages

### PR DESCRIPTION
**Pull request by ontowhee**
Fix for #1727 

I've removed the references to the mailing lists. I hope it looks good!

### Footer

Before
footer-mailing-list-before

After
footer-mailing-list-after

### Community

Before
community-mailing-list-before

After
community-mailing-list-after

### Download page

Before
download-mailing-list-before

After
download-mailing-list-after

### Homepage

Before
homepage-mailing-list-before

After
homepage-mailing-list-after

### Contact Foundation

Before
foundation-before

After
foundation-after

### Docs Template Sidebar

Before
sidebar-before

After
sidebar-after

### CODE REVIEW

File: djangoproject/templates/aggregator/index.html [hunk 1]
```diff
@@ -84,8 +84,4 @@ <h2 class="bullet-icon"><i class="icon icon-rss blue"></i> {{ feedtype.name }}</
       </li>
     {% endfor %}
   </ul>
-
-  <h2>{% translate "Mailing Lists" %}</h2>
-  {% include "includes/mailing_lists.html" %}
```
Comment (bmispelon) [inline djangoproject/templates/aggregator/index.html:89]: This included template is not used anymore now and can be deleted

File: djangoproject/templates/releases/download.html [hunk 2]
```diff
@@ -59,7 +59,7 @@ <h2>Option {% cycle options %}: Get the latest development version</h2>
 
   <h2>After you get it</h2>
   <p>See the <a href="{% url 'document-detail' lang='en' version='stable' url='intro/install' host 'docs' %}">installation guide</a> for further instructions. Make sure you read the documentation that corresponds to the version of Django you’ve just installed.</p>
-  <p>And be sure to sign up for the <a href="https://groups.google.com/group/django-users">django-users mailing list</a>, where other Django users and the Django developers themselves all hang out to help each other.</p>
+  <p>And be sure to sign up to the <a href="https://forum.djangoproject.com">Django Forum</a>, where other Django users and the Django developers themselves all hang out to help each other.</p>
```
Comment (bmispelon) [inline djangoproject/templates/releases/download.html:62]: Nice rewrite 👍🏻

**Review by bmispelon — CHANGES_REQUESTED** [review]
This is a great start, thanks for working on this 🙇🏻 

Using `git grep mailing.list` I found more places that should be fixed:
- [x] `djangoproject/templates/contact/foundation.html`
- [x] `docs/templates/docs/doc.html`

There's also things inside `djangoproject/templates/conduct/` that should be updated, but I don't know if that requires approval from the CoC work group. I'll ask around, we can do it in a separate PR later.

File: djangoproject/templates/aggregator/index.html (hunk 1 above)
Comment (ontowhee) [reply djangoproject/templates/aggregator/index.html:89]: Thanks for pointing this out! It is now deleted!

**Comment by ontowhee** [comment]
I've updated `djangoproject/templates/contact/foundation.html` and `docs/templates/docs/doc.html`. Thanks for mentioning these! I've added their screenshots to the PR description.

**Review by bmispelon — APPROVED** [review]
This is looking great, nice work 👍🏻 

Thanks for taking the time to take screenshots too, it made reviewing even easier 🙇🏻

**Comment by thibaudcolas** [comment]
Thank you! 😌 Now we have to figure out what to do with all that space on the homepage!

**Pull request merged**